types/polygon-lookup/package.json

```diff
@@ -7,7 +7,7 @@
     ],
     "dependencies": {
         "@types/geojson": "*",
-        "@types/rbush": "*"
+        "@types/rbush": "^3"
```

Comment:
The underlying library actually uses 2.x but they've been operating against 3.x for some time now, so I'm a little hesitant to change it. We could bump this down to 2.x and revert this change from the most recent 3.x if we wanted:

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70242/files#diff-a07aa4d367a37d9ab6a3042a6c8b663606c397c07601e490a570511f92b61f09R2

Reply:
Yeah, this range really should match the range in the underlying library. I do think it's probably worth fixing it up now, and dealing with any fallout in a more accurate way if necessary, but these are not super widely depended upon packages, so I feel like it will be fine.